Example/Tests/MiscTests.swift

```diff
@@ -73,7 +73,7 @@ class MiscTests: XCTestCase {
 
     func testNoKIDInHeaderJWT() {
         let options = [
-            "issuer": TestUtils.issuer,
+            "issuer": "https://example.oktapreview.com/oauth2/default",
```

Comment:
Why can't we use `TestUtils.issuer` here?

Reply:
Originally I couldn't find a reliable source for creating an RS256 `alg` JWT. Found one and updated the test function.